src/cmd/src/error.rs

```diff
@@ -210,7 +209,7 @@ pub enum Error {
     #[snafu(display("Failed to load layered config"))]
     LoadLayeredConfig {
         #[snafu(source)]
-        error: ConfigError,
+        source: Box<common_config::error::Error>,
```

Comment:
https://docs.rs/snafu/latest/snafu/derive.Snafu.html#transforming-the-source
This should get rid of `map_err(Box::new)`
```suggestion
        #[snafu(source(from(common_config::error::Error, Box::new)))]
        source: Box<common_config::error::Error>,
```